replicate.Rmd

```diff
@@ -0,0 +1,1684 @@
+
+```{r echo = FALSE}
+
+# low-income & loss gambling : attractive rating
+df3_spss %>%
+  filter(split == 0, smallloss == 1) %>%
+  summarise(mean.judge.lowincome.lossgamble = mean(judge), CI.low = mean(judge) - (1.96 * Sem(judge)), CI.high = mean(judge) + (1.96 * Sem(judge))) %>%
+  round(digits = 2)
+
+
+# low-income & no-loss gambling : attractive rating
+df3_spss %>%
+  filter(split == 0, smallloss == 0) %>%
+  summarise(mean.judge.lowincome.nolossgamble = mean(judge), CI.low = mean(judge) - (1.96 * Sem(judge)), CI.high = mean(judge) + (1.96 * Sem(judge))) %>%
+  round(digits = 2)
+
+```
+
+```{r echo = FALSE}
+df3_high_loss <- df3_spss %>%
+  filter(split == 1, smallloss == 1) %>%
+  summarise(mean.judge.highincome.lossgamble = mean(judge), CI = (1.96 * Sem(judge))) %>%
+  round(digits = 2)
+
+df3_high_noloss <- df3_spss %>%
+  filter(split == 1, smallloss == 0) %>%
+  summarise(mean.judge.highincome.nolossgamble = mean(judge), CI.low = (1.96 * Sem(judge))) %>%
+  round(digits = 2)
+#producing data.frames for new figure (3) - high income
+```
+
+```{r echo = FALSE}
+df3_low_loss <- df3_spss %>%
+  filter(split == 0, smallloss == 1) %>%
+  summarise(mean.judge.lowincome.lossgamble = mean(judge), CI = (1.96 * Sem(judge))) %>%
+  round(digits = 2)
+
+df3_low_noloss <- df3_spss %>%
+  filter(split == 0, smallloss == 0) %>%
+  summarise(mean.judge.lowincome.nolossgamble = mean(judge), CI = (1.96 * Sem(judge))) %>%
+  round(digits = 2)
+#producing data.frames for new figure (3) - low income
+```
+
+
+```{r echo = FALSE}
+title = factor(c("Lower Income", "Higher Income"), levels = c("Lower Income", "Higher Income"))
+values = c(df3_low_loss$mean.judge.lowincome.lossgamble, df3_high_loss$mean.judge.highincome.lossgamble, df3_low_noloss$mean.judge.lowincome.nolossgamble, df3_high_noloss$mean.judge.highincome.nolossgamble)
+gamble = c("loss_gamble", "loss_gamble", "no_loss_gamble", "no_loss_gamble")
+CIs = c(df3_low_loss$CI, df3_high_loss$CI, df3_low_noloss$CI, df3_high_noloss$CI)
+
+fig3.table <- data.frame(title, values, gamble, CIs)
+#figure 3 data frame
+fig3.table$gamble <- factor(fig3.table$gamble, levels = c("loss_gamble", "no_loss_gamble"))
+#relevel figure 3 table
+
+figure3 <- fig3.table %>%
+  ggplot(aes(x = title, y = values, fill = gamble)) + geom_bar(stat = "identity", position = position_dodge(), colour = "black") + ylab("Attractiveness of gamble") + xlab(NULL) + geom_errorbar(aes(ymin = values - CIs, ymax = values + CIs), width = 0.2, position = position_dodge(0.9)) + ggtitle("Figure 3 - Attractiveness of loss or no-loss gamble\nfor Study 3\n(error bars represent 95% confidence interval)") + theme_bw() + theme(panel.border = element_blank(), panel.grid.major = element_blank(), panel.grid.minor = element_blank(), axis.line = element_line(colour = "black"))
+#produces figure 3
+
+print(figure3)
+#displays figure 3
+```
```

Comment:
Is this called figure 3 in order to correspond with the tables in the original article?